packages/wine.rb

```diff
@@ -14,9 +14,17 @@ class Wine < Package
   })
 
   depends_on 'bison'
-  depends_on 'sommelier'
   depends_on 'flex'
   depends_on 'freetype'
+  depends_on 'ncurses'
+  depends_on 'cups'
+  depends_on 'gettext'
+  depends_on 'libgudev'
+  depends_on 'mesa'
+  depends_on 'krb5'
+  depends_on 'libxrandr'
```

Comment:
This might come along for the ride with sommelier.  Not sure but might be worth checking to avoid a potential dreaded dependency loop.

Reply:
No. It doesn't.